pkgs/os-specific/linux/igh-ethercat-master/ethercatctl.in.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/script/ethercatctl.in b/script/ethercatctl.in
+index 2da6e95b..7e560d6e 100755
+--- a/script/ethercatctl.in
++++ b/script/ethercatctl.in
+@@ -31,17 +31,17 @@
+ #
+ #------------------------------------------------------------------------------
+ 
+-LSMOD=/sbin/lsmod
+-MODPROBE=/sbin/modprobe
+-RMMOD=/sbin/rmmod
+-MODINFO=/sbin/modinfo
+-IP=/bin/ip
++LSMOD=/run/current-system/sw/bin/lsmod
++MODPROBE=/run/current-system/sw/bin/modprobe
++RMMOD=/run/current-system/sw/bin/rmmod
++MODINFO=/run/current-system/sw/bin/modinfo
++IP=/run/current-system/sw/bin/ip
```

Comment:
This should use paths from the packages directly and not ``/run/current-system``

Reply:
Fixed